caom2-tap/src/main/resources/CaomResolver.properties

```diff
@@ -0,0 +1,22 @@
+#
+# artifact resolver configuration
+#
+# scheme:<class name of SchemeHandler impl>
+
+# basic AD resolver
+ad=ca.nrc.cadc.caom2.artifact.resolvers.AdResolver
+# AD resolver that handles cutouts
+ad=ca.nrc.cadc.caom2.artifact.resolvers.AdResolverWithCutouts
```

Comment:
Since this file in in main/resources it will be inside the caom2-tap jar file and readable from the classpath. However, I do not see a CaomResolver class that is going to read this file and use it and I don't see any changes in CaomSchemeHandler to reflect the use of the StorageResolver interface or use of this properties file. So either there is a new CaomResolver class missing or this file is misnamed and there are some changes in CaomSchemeHandler missing...

Secondary issue: while it is good to document alternate implementations, there should be one configured resolver per scheme; in the case of the ad scheme, it should be the AdCutoutResolver and the normal AdResolver should be listed but commented out. Same for VOSpaceCutoutResolver.